frontend/src/lib/utils.tsx

```diff
@@ -559,11 +559,11 @@ export function someParentMatchesSelector(element: HTMLElement, selector: string
 }
 
 /** Convert camelCase to Title Case. Useful for generating page title from internal scene name. */
-export function camelCaseToTitle(camelCase: string): string {
+export function camelCaseToTitle(camelCase: string | number): string {
     const words: string[] = []
     let currentWord: string = ''
-    for (const character of camelCase.trim()) {
-        if (character == character.toLowerCase() && character != character.toUpperCase()) {
+    for (const character of String(camelCase).trim()) {
+        if (character === character.toLowerCase()) {
```

Comment:
Q: Not sure why the previous code was like that, do you know? 

Would be great if this code had any tests.

Reply:
It was like that because I just copied it from SO :) but it had a problem with numbers that way.